joomla/joomla-cms
# Adding support to the NEL header to the core httpheaders plugin

Pull Request for Issue https://github.com/zero-24/plg_system_httpheader/issues/43#issuecomment-1341606902

### Summary of Changes

Adding support to the NEL header to the core httpheaders plugin
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/NEL

### Testing Instructions

- Verify that you cant add a "NEL" header
- apply the patch
- confirm you can set the NEL header now

### Actual result BEFORE applying this Pull Request

No support for the NEL header

### Expected result AFTER applying this Pull Request

Now we do support the NEL header

### Link to documentations
Please select:
- [x] Documentation link for docs.joomla.org: https://docs.joomla.org/J4.x:Http_Header_Management
